### PR TITLE
Update badges issues #19

### PR DESCRIPTION
**Pull request by nelsonic**
Quick one just to tidy up. #19

### CODE REVIEW

**Review by SimonLab — APPROVED** [review]
LGTM :+1:

File: README.md [hunk 1]
```diff
@@ -6,13 +6,13 @@ Role Based Access Control (**`RBAC`**) gives you
 a human-friendly way of controlling access
 to specific data/features in your App(s).
 
-[![Build Status](https://img.shields.io/travis/com/dwyl/rbac/master.svg?style=flat-square)](https://travis-ci.com/dwyl/rbac)
+[![Build Status](https://img.shields.io/travis/dwyl/rbac/master.svg?style=flat-square)](https://travis-ci.org/dwyl/rbac)
```
Comment (nelsonic) [inline README.md:9]: derp! 🤦  I wasn't paying close enough attention last night. 😫 

image

new PR for the build badge: https://github.com/dwyl/rbac/pull/21

Comment (SimonLab) [reply README.md:9]: sorry should have spot that too

Comment (nelsonic) [reply README.md:9]: Not at all. it was a subtle thing and that's why we have badges to inform us. 😉

**Pull request merged**